crmd/remote_lrmd_ra.c

```diff
@@ -1173,4 +1189,43 @@ remote_ra_process_pseudo(xmlNode *xml)
         }
     }
     freeXpathObject(search);
+
```

Comment:
ah yes ... xml just holds one action and is  called repeatedly. So if it is already identified
as pseudo-fence we don't have to check. Was somehow assuming it might hold more
than one action ...